pkg/imagestore/imagestore.go

```diff
@@ -53,6 +54,8 @@ var DefaultVersions = []map[string]string{
 	},
 }
 
+var httpClient *http.Client
```

Comment:
If `downloadURLToFile`  is used only by `rhcosStore` does it make sense move this variable and function under `rhcosStore` somehow global doesn't feels right here.

Reply:
:+1: yes, making this part of the `rhcosStore` struct is what I had in mind

Reply:
Thanks for the clarification! I'll go ahead and move it under the `rhcosStore` struct